### PR TITLE
dev-games/ogre: Add latest release 1.10.11 and two testing ebuilds (1.11&2.1)

### PR DESCRIPTION
**Pull request by Yamakuzure**
Bug: https://bugs.gentoo.org/647266

- Add ebuild for the latest release v1.10.11
  This is the latest stable release of Ogre3D, released on 2017-12-31.

- Add snapshot ebuild for the 1.11 series.
  This is an ebuild for the last commit in the default branch from 2018-04-23.

- Add snapshot ebuild for the 2.1 series.
  This is an ebuild for the last commit in the v2-1 branch from 2018-04-20.
  The 2.1 series, although being the future of Ogre3D, still lacks a lot of the widely used components and plugins.
  However, developers who are starting their project now, like me, might be willing to accept these limitations in exchange for the chance to develop against the future API.

I have decided to not provide a true live ebuild, as there are many commits from which Ogre3D can not be built.
Although a build failure of a live package would only cause the user to wait and try again later, this render engine needs a considerable amount of compile time.
I will update the ebuild to a later commit (and revision), whenever I see significant commits being made, and can ensure, that the resulting source tree can be built.

Tested-by: Sven Eden yamakuzure@gmx.net
Closes: https://bugs.gentoo.org/647266

Package-Manager: Portage-2.3.28, Repoman-2.3.9

### CODE REVIEW

**Comment by a17r** [comment]
Thanks a lot for your work, and sorry this has taken a while. In the meantime, homepage was fixed to https and the switch to `x11-base/xorg-proto` happened, I'm going to fix on merge.

File: dev-games/ogre/ogre-1.10.11.ebuild [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+DESCRIPTION="Object-oriented Graphics Rendering Engine"
+HOMEPAGE="http://www.ogre3d.org/"
+SRC_URI="https://github.com/OGRECave/${PN}/archive/v${PV}.zip -> ${P}.zip"
+
+LICENSE="MIT public-domain"
+SLOT="0/1.10.0"
+KEYWORDS="~amd64 ~arm ~x86"
+
+IUSE="cg doc double-precision examples +freeimage gl3plus gles2 gles3 ois +opengl profile tools"
```
Comment (a17r) [inline dev-games/ogre/ogre-1.10.11.ebuild]: does it make sense to provide an opengl flag when the ebuild hard-depends on virtual/opengl anyway?

**Comment by a17r** [comment]
Here's what I did:
- Pushed your `1.10.11` bump
- Fixed a few things afterwards
- Kept your snapshots pending as they are, as I noticed `1.11.1` already released

File: dev-games/ogre/ogre-1.10.11.ebuild (hunk 1 above)
Comment (Yamakuzure) [reply dev-games/ogre/ogre-1.10.11.ebuild]: You are right. I went through all USE flags but missed that one. It really doesn't make any sense...

**Comment by Yamakuzure** [comment]
So if I understand you correctly, I can advance to 1.11.1 with this PR, and can do better this time, right?

I'll see to it that your fixes will be transported to the new release.

**Comment by a17r** [comment]
Exactly, you can keep using this PR after rebase or instead open a new one to make it more visible, up to you.

**Pull request closed without merging**